Lombiq.Walkthroughs/Assets/Scripts/walkthroughs.js

```diff
@@ -0,0 +1,3770 @@
+                // Tour id should be the same as the tour variable's name.
+                id: 'orchardCoreAdminWalkthrough',
+                useModalOverlay: true,
+                defaultStepOptions: {
+                    cancelIcon: {
+                        enabled: true,
+                    },
+                    when: {
+                        show() {
+                            addShepherdQueryParams();
+                        },
+                    },
+                },
+                steps: [
+                    {
+                        title: 'Welcome in the<br>Orchard Core Admin Walkthrough!',
+                        text: 'This walkthrough covers key Orchard Core features, such as content management, user roles, ' +
+                            'and theme selection, and points users to further learning resources.',
+                        buttons: [
+                            {
+                                action: function () {
+                                    removeShepherdQueryParams();
+                                    walkthroughs.orchardCoreAdminWalkthrough.complete();
+                                    // The walkthroughSelector is later defined and orchardCoreAdminWalkthrough is only used later.
+                                    // eslint-disable-next-line no-use-before-define
+                                    walkthroughSelector.start();
+                                },
+                                classes: 'shepherd-button-secondary',
+                                text: 'Back',
+                            },
+                            nextButton,
+                        ],
+                        id: 'welcome',
+                    },
+                    {
+                        title: 'Setup recipe',
+                        text: 'The setup recipe in <a href="https://github.com/Lombiq/Orchard-Walkthroughs">Lombiq.Walkthroughs module</a>' +
+                            '(named Walkthroughs) used the <a href="https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json">Blog recipe</a>' +
+                            ' as a base recipe. In Orchard Core, a <a href="https://docs.orchardcore.net/en/main/docs/reference/modules/Recipes/">recipe</a> ' +
+                            'is a json file that defines a set of instructions for setting up and configuring an' +
+                            ' Orchard Core application. Recipes can include predefined content types, widgets, menus, content items, and other' +
+                            'configurations.They are used to streamline the setup of an Orchard Core site, making it easier ' +
+                            '  to create consistent site structures and content. Recipes can be executed during the initial' +
+                            ' setup of a site or at any point to apply configurations or import content.',
```

Comment:
Please check and fix all these multiline texts and follow the same pattern everywhere, thus you won't have these multi-spaces or missing spaces in the text.
E.g. have the needed space at the end of the line always and don't use a space at the beginning of the new line.